cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/interceptors/NGSINameMappingsInterceptor.java

```diff
@@ -384,7 +384,10 @@ protected void loadNameMappings(String jsonStr) {
             LOGGER.debug("[nmi] Entity found: " + originalEntityId + ", " + originalEntityType);
 
             if (entityMapping.getNewEntityId() != null) {
-                newEntityId = entityMapping.getNewEntityId();
+                LOGGER.debug("[nmi] IdPattern : " + entityMapping.getOriginalEntityIdPattern().toString());
```

Comment:
Nice extension. Anyway, are you thinking only in entity IDs? Why not extending this feature to all the new values, i.e. new service, new service path, new entity type, new attribute name and new attribute type?

Reply:
We had thought about extending it to all values, but finnaly discarted because those values usually are "constats" in our production environments and we can´t find any example needing it. 

Anyway, it will be easily done if needed.

Reply:
I think it is enough for now if this is added as a techdebt issue.

Reply:
This is the issue https://github.com/telefonicaid/fiware-cygnus/issues/1458